From 992838370030056515d21704c6f0044d65f03c7b Mon Sep 17 00:00:00 2001
From: Marian Chereji <marian.chereji@freescale.com>
Date: Mon, 21 May 2012 23:07:26 +0000
Subject: [PATCH 200/518] dpa_classifier: Remove unused variable

Removed unused variable "fm_pcd" from the function
"table_insert_entry_exact_match".

Signed-off-by: Marian Chereji <marian.chereji@freescale.com>
[Grabbed from the branch, LINUX_IR5.2.0, of
https://git.freescale.com/git-private/cgit.cgi/ppc/alu-b4860/linux.git.]
Signed-off-by: Tiejun Chen <tiejun.chen@windriver.com>
---
 drivers/staging/fsl_dpa_offload/dpa_classifier.c |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/fsl_dpa_offload/dpa_classifier.c b/drivers/staging/fsl_dpa_offload/dpa_classifier.c
index c7ad427..a37da7c 100644
--- a/drivers/staging/fsl_dpa_offload/dpa_classifier.c
+++ b/drivers/staging/fsl_dpa_offload/dpa_classifier.c
@@ -1693,7 +1693,6 @@ static int table_insert_entry_exact_match(struct dpa_cls_table	*cls_table,
 	uint8_t shadow_table_index;
 	uint8_t key_data[DPA_CLS_TBL_MAXENTRYKEYSIZE];
 	uint8_t mask_data[DPA_CLS_TBL_MAXENTRYKEYSIZE];
-	t_Handle fm_pcd;
 	struct dpa_cls_tbl_shadow_table *shadow_table;
 	struct dpa_cls_tbl_entry *index_entry;
 	struct list_head *list_current;
@@ -1801,7 +1800,6 @@ static int table_insert_entry_exact_match(struct dpa_cls_table	*cls_table,
 				list_current = &cls_table->entry_list;
 	}
 
-	fm_pcd = (t_Handle)cls_table->params.fm_pcd;
 	if ((i > 0) && (!cls_table->int_cc_node[i].used)) {
 		/* Check whether this is the last used node */
 		for (j = i; j < cls_table->int_cc_nodes_count; j++) {
-- 
1.7.5.4

